### DISCUSSION TITLE
Redirect URL in google-auth [Jul 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3034] [Discord Threads]
> 🙋‍♂️ **uma**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### ya some of our users has integrated with google-auth    
#### for setting the URI - you would have to set certain env variables like NC_PUBLIC_URL   
#### https://docs.nocodb.com/getting-started/installation#environment-variables  
#  
> 🙋‍♂️ **uma**  
#### thank you - works great!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1002318784981373049)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
